From 68cc945d17b1c6ea1216a8edcb76761fe92562bf Mon Sep 17 00:00:00 2001
From: Takeshi Saito <takeshi.saito.xv@renesas.com>
Date: Wed, 20 Mar 2019 18:15:13 +0900
Subject: [PATCH 787/909] arm64: dts: salvator-common: Add no-sdio and no-sd
 properties to SDHI2

commit 5c758f337e7f511d9ba92d0885d43fe7bc7d7557 from
git://git.kernel.org/pub/scm/linux/kernel/git/horms/renesas-bsp.git

Add the 'no-sdio' and 'no-sd' properties to the SDHI2 node used as the
eMMC port on Salvator board. It will limits the send sdio and sd cmd
during mmc initialization.

In addition, the 'no-sdio' property uses multiple SG entries when SDHI
internal DMAC with IOMMU is enabled. This can improve performance.

Signed-off-by: Takeshi Saito <takeshi.saito.xv@renesas.com>
Signed-off-by: Takeshi Kihara <takeshi.kihara.df@renesas.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm64/boot/dts/renesas/salvator-common.dtsi | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/arch/arm64/boot/dts/renesas/salvator-common.dtsi b/arch/arm64/boot/dts/renesas/salvator-common.dtsi
index 922b9cc4797c..45489ec9a1c2 100644
--- a/arch/arm64/boot/dts/renesas/salvator-common.dtsi
+++ b/arch/arm64/boot/dts/renesas/salvator-common.dtsi
@@ -833,6 +833,8 @@
 	bus-width = <8>;
 	mmc-hs200-1_8v;
 	mmc-hs400-1_8v;
+	no-sd;
+	no-sdio;
 	non-removable;
 	fixed-emmc-driver-type = <1>;
 	status = "okay";
-- 
2.17.1

